state-chain/runtime/src/runtime_apis.rs

```diff
@@ -164,7 +164,7 @@ decl_runtime_apis!(
 		fn cf_min_swap_amount(asset: Asset) -> AssetAmount;
 		fn cf_max_swap_amount(asset: Asset) -> Option<AssetAmount>;
 		fn cf_min_deposit_amount(asset: Asset) -> AssetAmount;
-		fn cf_prewitness_swaps(from: Asset, to: Asset) -> Option<Vec<AssetAmount>>;
```

Comment:
Again the from and to here are meaningful. As in the old rpcs they do implicitly describe the direction.

Reply:
After some thoughts, I think it is simpler to use (from, to) than (base, quote, side), because the first is more obvious and straight forward. 
What do you think ? whats your thought/preference on this?

Reply:
Ok, I think we can leave this rpc with "from" and "to", but in the future add a new one with "base", "quote" and "side". The reason I'd like to move to suing "side" in the new rpcs, is because we had this exact problem with LPs not understanding the order of the two parameters mattered, and having a "side" arguments makes to very clear.